packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -642,19 +642,6 @@ contract VotingV2 is
      *        VOTING GETTER FUNCTIONS       *
      ****************************************/
 
-    /**
```

Comment:
I moved this function to `Staker` because all the state `delegateToStaker` and `voterStakes` is in the latter contract.